capture/README.md

```diff
@@ -0,0 +1,44 @@
+# OT-SCA Capture Scripts
+
+The `caputure_*.py` scripts propvided in this directory allow the user to
+capture traces of a cryptographic operation performed by OpenTitan.
+
+## Quick Usage
+
+Follow the steps described in the [`getting_started`](../doc/getting_started.md)
+documentation to setup the environment, the target, and the scope.
+
+When using OpenTitan on the CW310 and measuring traces using Husky, AES traces
+for a random key tests can be captured with the following command:
+```console
+mkdir -p projects/
```

Comment:
nit: i think you dont even need to mkdir

Reply:
If you want to store the traces in projects/ you need to make sure that the dir. exists as the script is not creating a directory.